### PR TITLE
added starter notebook for reverse training

### PR DESCRIPTION
**Pull request by CateGitau**
Here's the reverse training starter notebook/template. Everything is the same as the starter notebook, I just changed the config file such that the src now beomes the tgt and vice versa.

### CODE REVIEW

File: starter_notebook_reverse_training.ipynb [hunk 1]
```diff
@@ -0,0 +1,1292 @@
+{
```
Comment (juliakreutzer) [inline starter_notebook_reverse_training.ipynb:1]: <p>A big visible comment would help to clarify the purpose of this notebook compared to the starter one. We can also rename this one as "into_English" for example, and the old starter notebook "from_English".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (juliakreutzer) [inline starter_notebook_reverse_training.ipynb:1]: <p>The gdrive_path should probably contain "reverse" as well, or have src and trg flipped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (juliakreutzer) [inline starter_notebook_reverse_training.ipynb:1]: <p>Please add a comment here that the reversal is on purpose ;)</p><p>The name variable can directly be changed to $trg_$src.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by juliakreutzer** [comment]
Thanks a lot @CateGitau! I wonder how to make it most clear to newcomers what "reverse" means, and how to explain that the data is en-xx focused, but the model can be trained with the same data in both directions. 
Reviewing the notebook I realized that we should probably refactor it (e.g. separate data prep from training, or rename src and trg for JW300 downloads to "en" and "xx" or the like), and add a few more comments here and there. This is not relevant for this PR, but for the future :)

File: starter_notebook_reverse_training.ipynb (hunk 1 above)
Comment (CateGitau) [reply starter_notebook_reverse_training.ipynb:1]: Noted will change that asap

Comment (CateGitau) [reply starter_notebook_reverse_training.ipynb:1]: Alright Noted

Comment (CateGitau) [reply starter_notebook_reverse_training.ipynb]: flipped src and trg

Comment (CateGitau) [reply starter_notebook_reverse_training.ipynb]: Didn't quite get this but I added a note at the beginning:)

**Review by juliakreutzer — CHANGES_REQUESTED** [review]
Looks good! The only thing left is the .ipynb_checkpoints directory which needs to be excluded from the commit.

**Comment by CateGitau** [comment]
> Looks good! The only thing left is the .ipynb_checkpoints directory which needs to be excluded from the commit.

done! removed them

**Review by juliakreutzer — APPROVED** [review]
thanks for the renaming!

**Comment by CateGitau** [comment]
Resolved conflicts

**Comment by juliakreutzer** [comment]
Awesome, thanks so much!! :100:

**Pull request merged**